node/network/bridge/src/multiplexer.rs

```diff
@@ -68,19 +70,31 @@ impl RequestMultiplexer {
 			})
 			.unzip();
 
+		// Ok this code is ugly as hell, it is also a hack, see https://github.com/paritytech/polkadot/issues/2842.
+		// But it works and is executed on startup so, if anything is wrong here it will be noticed immediately.
 		let index = receivers.iter().enumerate().find_map(|(i, (p, _))|
 			if let Protocol::StatementFetching = p {
 				Some(i)
 			} else {
 				None
 			}
-		).expect("Statement fetching must be registered. qed.");
+		).expect("Statement fetching must be registered.");
```

Comment:
"prove or remove" - looks like the 'QED' disappeared?

Reply:
Yeah - it is not really proven, but rather obvious. I think it is fine for the following reasons:
1. It is true.
2. If it were not, this code would panic immediately on startup, so it is impossible to hit production, if we do any testing at all.
3. It is just a hack, until I will finally get rid of this multiplexer all together, see
#2842
and
#3432

Reply:
I think @rphmeier refers to the style, specifically `; qed` at the end

Reply:
Sure, I can add it back :-)